src/main/java/racingcar/controller/RacingGameController.java

```diff
@@ -2,27 +2,50 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Random;
-import racingcar.domain.DefaultRandomService;
 import racingcar.domain.NumberOfAttempts;
 import racingcar.domain.PositiveNumber;
 import racingcar.domain.RacingCar;
 import racingcar.domain.RacingCars;
-import racingcar.domain.RandomService;
-import racingcar.dto.TotalRacingGameResult;
+import racingcar.dto.GameResultInfo;
+import racingcar.dto.RacingCarDto;
 import racingcar.service.RacingCarService;
 import racingcar.view.OutputView;
-import racingcar.view.WinnerUtils;
 
 public class RacingGameController {
+    private final RacingCarService racingCarService;
     private final OutputView outputView;
 
-    public RacingGameController(OutputView outputView) {
+    public RacingGameController(RacingCarService racingCarService, OutputView outputView) {
         this.outputView = outputView;
+        this.racingCarService = racingCarService;
     }
 
     public void run(List<String> carNames, PositiveNumber numberOfAttempts) {
-        startAllGames(createRacingCars(carNames), createNumberOfAttempts(numberOfAttempts));
+        informWinners(startAllGames(createRacingCars(carNames), createNumberOfAttempts(numberOfAttempts)));
+    }
+
+    private void informWinners(List<RacingCarDto> finalGameResult) {
+        outputView.printWinners(createRacingCarsWith(finalGameResult).findWinners());
+    }
+
+    private RacingCars createRacingCarsWith(List<RacingCarDto> finalGameResult) {
+        List<RacingCar> racingCars = new ArrayList<>();
+        for (RacingCarDto racingCarDto : finalGameResult) {
+            racingCars.add(new RacingCar(racingCarDto.getCarName(), racingCarDto.getPosition()));
+        }
+        return new RacingCars(racingCars);
```

Comment:
Stream에 대해 참고해보시면 좋을것 같습니다.

아래와 같은 방식으로도 표현할 수 있기 때문입니다.

for-loop는 반복되는 과정을 위해 코드를 위, 아래로 읽으며 파악해야하지만 Stream을 이용하여 표현하면 아래로만 읽을수 있는 장점이 있습니다.

혹시 오해하실까 말씀드리자면 Stream이 무조건 더 낫다는 것은 아닙니다.

```suggestion
        return finalGameResult.stream()
                .map(it -> new RacingCar(it.getCarName(), it.getPosition()))
                .collect(Collectors.collectingAndThen(Collectors.toList(), RacingCars::new));
```